index.html

```diff
@@ -0,0 +1,173 @@
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="normalize.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet"> 
+    <title>Juan Pablo Juguera - Portfolio</title>
+</head>
+
+<body>
+
+    <nav id="navbar">
+        <div class="nav__bar">
+            <i class="fa-solid fa-bars" onclick="hideMenu()"></i>
+            <div class="nav__links" id="id__links">
+                    <a class="nav__link" href="#home">Home</a>
+                    <a class="nav__link nav__link-color" href="#about">About me</a>
+                    <a class="nav__link" href="#projects">Projects</a>
+                    <a class="nav__link nav__link-color" href="#contact">Contact</a>       
+            </div>
+        </div>
+    </nav>
+
+    <header class="home" id="home">
+        <div class="home__profile">
+            <img class="profile__img" src="Pictures/Programador.png" alt="">
+            <div class="profile__text">
+                <div class="profile__title">Hi!</div>
+                <h1 class="profile__name">I'm Juan Pablo Juguera</h1>
+                <div class="profile__title">FRONTEND DEVELOPER</div>
+            </div>
+        </div>
+        
+        <p class="profile__location">Based in Buenos Aires , Argentina</p>
+
+        <a class="download__button" href="Pictures/cv.eps" download>
+                <i class="fa-solid fa-arrow-down"></i>
+                <p class="download__text">Download CV</p> 
+        </a>
+    </header>
+
+    <div class="section__div" id="about"> </div>
+
+    <section class="about">
+        <h2 class="about__title">ABOUT ME</h2>
+        <h3 class="about__subtitle">SKILLS</h3>
+
+        <div class="about__skills">
+            <img class="about__skill" src="Pictures/html-icon.png" alt="HTML">
+            <img class="about__skill" src="Pictures/CSS-icon.png" alt="CSS">
+            <img class="about__skill" src="Pictures/js-icon.png" alt="JS">
+            <img class="about__skill" src="Pictures/git2-icon.png" alt="GIT">
+            <img class="about__skill" src="Pictures/sass-icon.png" alt="SASS">
+        </div>  
+
+        <h3 class="about__hobbies">HOBBIES</h3>
+        <div class="hobbies">
+            <div class="hobbie">
+                <img class="hobbie__img" src="Pictures/gym-icon.png" alt="gym">
+                <h3 class="hobbie__title">Gym</h3>
+                <p class="hobbie__desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus nostrum similique quia dolores deleniti voluptate provident neque, blanditiis tempora ad consequ.</p>
+            </div>
+
+            <div class="hobbie">
+                <img class="hobbie__img" src="Pictures/gaming-icon.png" alt="gaming">
```

Comment:
No deberias de cerrar el tag?
```suggestion
                <img class="hobbie__img" src="Pictures/gaming-icon.png" alt="gaming" />
```